README.md

```diff
@@ -174,6 +174,52 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
 ## Getting Help
 * We need to setup a helpdesk!!
 
+## Parallel and Distributed Processing
+
+### Terminology
+
+* CPU
+  * Processing chip. Quality and number of CPU determines compute power -- the rate at which computations can be performed.
+* Distributed Processing
+  * Running a set of processes/jobs by farming them out to a network of compute nodes/machines. Many of these jobs will run in parallel. (Distinction between this and "parallel processing" is not deeply important here.)
+* Node
+  * A single machine within a network of machines used for distributed processing.
+* Parallel Processing
+  * Running >1 process/job concurrently. (Distinction between this and "distributed processing" isn't deeply important here, but can confusing.)
+* RAM
+  * Working memory in a machine/node. Amount of RAM determines how much data can be loaded in-memory simultaneously.
+* Worker
+  * An entity that completes a chunk of work (data + instructions). It runs in the background.
+
+### When to use distributed or parallel processing?
+
+1. Your dataset is very large, but could be split into subsets that can be processed individually.
+  * If some of your processing steps depend on information about the full dataset, consider whether you can separate those steps from the ones that don't. Then do the latter in parallel/distributed. Either compute the full-dataset info first and pass it in as function args, or collect the results from parallel processing and then compute the full-dataset info.
+ 
+2. Your computations/operations take a long time, but can be split into subsets that are independent from each other (the input to one does not depend on the output of another).
+  * It is unlikely that every step in your pipeline is independent of the input/output of the others. But, consider whether there is at least one subset within which the steps are independent of each other -- that subset can be split and run in parallel.
+
+### Distributed processing with Dask DataFrames -- basic concepts
+
+The Fornax platform will primarily use Dask for distributed processing.
```

Comment:
I would suggest we delete this line, and add it back in when it is true.  I expect the documentation to go live before the majority of our notebooks use Dask.  I still think it is useful to have some basic concepts on Dask in the documentation.